common/ideologies/cwp_00_leader_communist_ideologies.txt

```diff
@@ -95,125 +107,335 @@ ideology_authoritarian = {
+		law_anocracy = neutral
+		law_oligarchy = strongly_approve
+		law_autocracy = disapprove
+		law_representative_democracy = disapprove
+		law_consensus_democracy = disapprove
+		law_direct_democracy = strongly_disapprove
 	}
-	lawgroup_nationalization = {
-		law_total_nationalization = strongly_approve
-		law_heavy_industry_nationalization = approve
-		law_military_industry_nationalization = neutral
-		law_resource_extraction_nationalization = neutral
-		law_public_utility_companies = disapprove
-		law_no_nationalization = strongly_disapprove
+	lawgroup_criminal_justice = {
+		law_arbitrary_justice = disapprove
+		law_political_incarceration = approve
+		law_labor_camps = neutral
+		law_penal_correction = approve
+		law_criminal_rehabilitation = neutral
+		law_criminal_restitution = strongly_disapprove
 	}
-	lawgroup_economic_system = {
-		law_socialization = strongly_approve
-		law_interventionism = neutral
-		law_free_market = disapprove
-		law_anti_industrialization = strongly_disapprove
-		law_decentralization = strongly_disapprove
+	lawgroup_policing = {
+		law_militarized_police = approve
+		law_secret_police = approve
+		law_professional_police = neutral
+		law_privatized_police = strongly_disapprove
+		law_morality_police = disapprove
+		law_decentralized_police = strongly_disapprove
+	}
+	lawgroup_free_speech = {
+		law_outlawed_dissent = neutral
+		law_censorship = strongly_approve
+		law_right_of_assembly = disapprove
+		law_protected_speech = strongly_disapprove
 	}
 	possible = {
-		scope:interest_group = {
-			or = {
-				has_ideology = ideology:ideology_vanguardist
-				has_ideology = ideology:ideology_apparatus
-				is_interest_group_type = ig_ruralists
-				is_interest_group_type = ig_securocrats
+		and = {
+			scope:interest_group = {
+				or = {
+					is_interest_group_type = ig_bureaucrats
+					is_interest_group_type = ig_securocrats
+					is_interest_group_type = ig_laborists
+					
+				}
+			}
+			owner = {
+				has_law = law_type:law_socialization
+				has_law = law_type:law_council_elective
+			}
+			nor = {
+				has_law = law_type:law_representative_democracy
+				has_law = law_type:law_consensus_democracy
+				has_law = law_type:law_direct_democracy
 			}
 		}
 	}
 	leader_weight = {
-		value = 100
+		value = 500
 		if = {
 			limit = {
 				scope:interest_group = {
-					is_interest_group_type = ig_ruralists
+					is_interest_group_type = ig_bureaucrats
+				}
+			}
+			add = 200
+		}
+		else_if = {
+			limit = {
+				scope:interest_group = {
+					is_interest_group_type = ig_securocrats
+				}
+			}
+			add = 100
+		}
+		else_if = {
+			limit = {
+				scope:interest_group = {
+					is_interest_group_type = ig_laborists
+				}
+			}
+			add = 50
+		}
+		else_if = {
+			limit = {
+				owner = {
+					has_law = law_type:law_oligarchy
+				}
+			}
+			add = 100
+		}
+	}
+}
+
+ideology_reformist = {
+	icon = "gfx/interface/icons/ideology_icons/ideology_leader/ideology_leader_no_ideology.dds"
```

Comment:
stop using no ideology. it's confusing. pick a real icon.

Reply:
oki